dbt-snowflake/src/dbt/adapters/snowflake/impl.py

```diff
@@ -65,6 +65,7 @@ class SnowflakeAdapter(SQLAdapter):
 
     AdapterSpecificConfigs = SnowflakeConfig
 
+    CATALOG_INTEGRATIONS = CATALOG_INTEGRATIONS
```

Comment:
why reassign the constant?

Reply:
I'm assigning the value `dbt.adapters.snowflake.catalogs.CATALOG_INTEGRATIONS` to the class variable `SnowflakeAdapter.CATALOG_INTEGRATIONS`. They just have the same name. I can update the name if it's confusing.

Reply:
Yes please. This context will be lost after this PR is merged.

Reply:
I'm going to move the assignment here and pull it from the `catalogs` package. It's sort of misplaced anyway.